united-fare-finder.py

```diff
@@ -36,45 +36,96 @@
 elem = driver.find_element(By.ID, "closeBtn")
 elem.click()
 
-elem = driver.find_element(By.ID, "bookFlightOriginInput")
-elem.clear()
-elem.send_keys("ORD")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(By.ID, "bookFlightDestinationInput")
-elem.clear()
-elem.send_keys("HNL")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(By.ID, "DepartDate")
-elem.clear()
-elem.send_keys("03/03/23")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(By.ID, "ReturnDate")
-elem.clear()
-elem.send_keys("03/10/23")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(
-  By.XPATH, '//*[@id="bookFlightForm"]/div[1]/div[1]/label')
-elem.click()
-
-elem = driver.find_element(
-  By.CSS_SELECTOR,
-  "button.app-components-BookFlightForm-bookFlightForm__findFlightBtn--1lbFe")
-elem.submit()
-
-time.sleep(10)
-
-# scrape fare data
-page_source = driver.page_source
-flight_elements = parsePageSource(page_source)
-
-# write to file
-for i in range(len(flight_elements)):
-    for j in range(len(flight_elements[i])):
-        print(*flight_elements[i][j])
-    print()
+origin = "ORD"
+dest = [
+  "hnl", "slc", "dfw", "lax", "sea", "atl",
+  "mia", "mco", "sfo", "jfk", "ewr", "lga", "dca", "iad"
+]
+depDate = "03/01/2023"
+retDate = "03/10/2023"
+
+count = 0
+for dest in dest:
+
+    # input flight info in main page
+    if (count == 0):
```

Comment:
I think its cleaner to use `index` with `enumerate` here
https://stackoverflow.com/questions/522563/accessing-the-index-in-for-loops

Reply:
I actually changed this to a boolean right after I initiated the pull request.  I wasn't trying to index/count anything.  I was trying to trigger the if statement.  :)

But noted, I'll need to know this at some point.